src/architecture/configuration.md

```diff
@@ -92,17 +109,42 @@ The configuration used to create a genesis block.
 - **tcp_reconnect_timeout_max**  
   Maximum timeout (ms) for reconnect attempt.
 
-#### [node.api]
+#### [api]
 
 API configuration parameters.
 
 - **enable_blockchain_explorer**  
   Enable api endpoints for the blockchain explorer on the public API address.
+- **state_update_timeout**  
+  Timeout (ms) to update API state.
```

Comment:
The description is unclear. Is updating the API state covered anywhere in documentation? If not, please add a sentence or two describing what the variable means.